deploy/kubeturbo_yamls/turbo_kubeturboDeploy_RBAC_usig_credentials_secret_sample.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  # use this yaml to deploy the kubeturbo pod
+  # Provide a value for the deploy/pod name: and update namespace if needed
+  name: kubeturbo
+  namespace: turbo
+  labels:
+    app: kubeturbo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubeturbo.io/monitored: "false"
+      labels:
+        app: kubeturbo
+    spec:
+      # Update serviceAccount if needed
+      serviceAccount: turbo-user
+      containers:
+        - name: kubeturbo
+          # Replace the image with desired version. Refer to readme in deploy for more details
+          image: turbonomic/kubeturbo:6.4.0
+          imagePullPolicy: IfNotPresent
+          args:
+            - --turboconfig=/etc/kubeturbo/turbo.config
+            - --v=2
+            # Uncomment the following two args if running in Openshift or k8s 1.11+
+            - --kubelet-https=true
+            - --kubelet-port=10250
+            # Uncomment to stitch using IP, or if using Openstack, Hyper-V/VMM
+            #- --stitch-uuid=false
+          volumeMounts:
+            # volume will be created, any name will work and must match below
+            - name: turbo-volume
+              mountPath: /etc/kubeturbo
+              readOnly: true
+            - name: opsmgr-credentials-volume
+              mountPath: /etc/opsmgr-credentials
```

Comment:
if we are using volume mounts, can you please make this optional: true

Reply:
Done!
Also updated the same in the existing deployment sample and removed the additional deployment sample.